README.md

```diff
@@ -51,6 +51,30 @@ Set of datasets to bootstrap a working dialogue model for human interaction
 - Uses zmq to talk to other toolboxes not in Python, examples of Lua Torch given.
 - Supports hogwild and batch training of models.
 
+## Requirements
```

Comment:
We want to explain how easy to use it is first, and how it works.
We don't want installation at the top of the README, we want to tell people what it's about first, see e.g. https://github.com/pytorch/pytorch/blob/master/README.md

We should move installation way down the file.